.claude/rules/design.md

```
@@ -0,0 +1,406 @@
+label: "What is your National Insurance number?",
+hint: "It's on your National Insurance card, benefit letter, payslip or P60. For example, 'QQ 12 34 56 C'."
+
+❌ AVOID: Vague, complex question
+
+Personal details
+We need some information about you
+
+### Error Handling Pattern
+
+Error Summary (required at top of page): 
+  - Title: "There is a problem"
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)